yourgov/components/FormMobileFoodVendorPermit/UploadFileTable.tsx

```diff
@@ -0,0 +1,212 @@
+
+type UploadFileTableProps = {
+	/** The id of the heading that describes the table. */
+	headingId?: string;
+	/** The id of the table. */
+	tableId?: string;
+	/** Open drawer. */
+	openDrawer: (document: Document) => void;
+	/** Remove an uploaded file. */
+	removeFile: (document: Document) => void;
+	/** Data for upload documents table. */
+	documents: Document[];
+};
+
+export const UploadFileTable = forwardRef<
+	HTMLTableElement,
+	UploadFileTableProps
+>(function UploadFileTable(
+	{ documents, headingId, openDrawer, removeFile, tableId },
+	ref
+) {
+	const error = false;
+	const loading = false;
+
+	if (error) {
+		return (
+			<Stack gap={1}>
+				<AlertFilledIcon color="error" size="lg" />
+				<Heading type="h2" fontSize="lg">
+					Failed to load
+				</Heading>
+				<Text>
+					There was an error loading the data. Please try refreshing the page to
+					try again.
+				</Text>
+			</Stack>
+		);
+	}
+
+	return (
+		<Stack gap={0.5}>
+			<TableWrapper>
+				<Table
+					aria-rowcount={10}
+					{...(headingId && {
+						'aria-labelledby': headingId,
+						'aria-describedby': descriptionId,
+					})}
+					id={tableId}
+					ref={ref}
+					tabIndex={-1}
+					tableLayout="fixed"
+				>
+					<TableHead>
+						<TableRow aria-rowindex={1}>
+							{headers.map(({ label, textAlign, width }) => {
+								return (
+									<TableHeader
+										key={label}
+										scope="col"
+										textAlign={textAlign}
+										width={width}
+									>
+										{label}
+									</TableHeader>
+								);
+							})}
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{loading ? (
+							<Fragment>
+								{Array.from(Array(10).keys()).map((i) => (
+									<TableRow key={i}>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonBox height={32} />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+									</TableRow>
+								))}
+							</Fragment>
+						) : (
+							<Fragment>
+								{documents.map((document, index) => {
+									const { id, documentType, error, file, size } = document;
+									// Adding 2 because the table header row is the first row
+									const rowIndex = index + 2;
+									return (
+										<TableRow
+											aria-rowindex={rowIndex}
+											invalid={error}
+											key={documentType}
+										>
+											<TableCell as="th" scope="row">
+												{documentType}
+											</TableCell>
+											<TableCell>
+												<Box>{file}</Box>
```

Comment:
Need to word break 
<img width="584" alt="image" src="https://github.com/user-attachments/assets/90c3c525-dc1b-4eaf-b1ff-dba8d64fbb3f">